code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -19,6 +19,7 @@
 	disliked_food = GRAIN | DAIRY
 	liked_food = GROSS | MEAT
 	ass_pic = "unathi"
+	examine_text = "<font color='#3d0000'> a Unathi</font>"

```

Comment:
an

Reply:
The only codebase out there that has a sound file for Unathi is Paradise. It's "Yu-nathi" there, so that'd be an "a". >:C